### PR TITLE
Rename TransactionApplied to IncludedTransaction

### PR DESCRIPTION
**Pull request by NunoAlexandre**
Closes https://github.com/radicle-dev/radicle-registry/issues/455

### CODE REVIEW

**Comment by xla** [comment]
For consideration https://github.com/radicle-dev/radicle-registry/issues/455#issuecomment-630506917

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,10 @@ Upcoming
 
 ### Breaking changes
 
+* Rename `TransactionApplied` to `IncludedTransaction`
 * cli: Rename the key-pair storage file from 'accounts.json' to 'key-pairs.json'
 * cli: Move key-pair related commands under the new `key-pair` command group
-* client: `TransactionApplied` result is now `Result<(), TransactionError>`
+* client: `IncludedTransaction` result is now `Result<(), TransactionError>`
```
Comment (geigerzaehler) [inline CHANGELOG.md:12]: Does it make sense to retroactively change the change log? I think this misrepresents the change

Comment (NunoAlexandre) [reply CHANGELOG.md:12]: No, not at all, my bad.

**Comment by NunoAlexandre** [comment]
> I would strongly suggest to use `TransactionIncluded` instead of `IncludedTransaction`. With the latter I would expect an `IncludedTransaction` to also be usable as a transaction. The adjective says basically “an included transaction is also a transaction”. But the struct does not reflect that: An included transaction only holds the hash of the transaction it references. IT cannot be used as a transaction.
> 
> On the other hand `TransactionIncluded` makes it clearer that the data is a representation of the fact that a transaction has been included. This also matches the documentation of the item much better: “Indicator that a transaction has been included in a block...”

I see your point. To be honest, I always struggled with `TransactionApplied` but I see your argument. To be the fairest to your point, `IncludedTxData` would be the clearest, as that is _exactly_ what the struct stands for: data relative to a transaction that was included in a block.

Thoughts?

**Comment by NunoAlexandre** [comment]
@geigerzaehler regarding my comment above, let's now spend time there, I pushed a commit renaming the struct and all variables accordingly.

**Pull request merged**